fhttp/http_test.go

```diff
@@ -1120,6 +1129,11 @@ func TestDebugHandlerSortedHeaders(t *testing.T) {
 	o.AddAndValidateExtraHeader("CCC: ccc")
 	o.AddAndValidateExtraHeader("ZZZ: zzz")
 	o.AddAndValidateExtraHeader("AAA: aaa")
+	// test that headers usually Add (list) but stay in order of being set
```

Comment:
```suggestion
	// Test that headers usually Add (list) but stay in order of being set.
```

Reply:
See my general comment, this is as an example

Reply:
confirmed the linter for that still works:

removed period in one of the file:
```
fhttp/http_client.go:41:75: Comment should end in a period (godot)
// Fetcher is the Url content fetcher that the different client implements
                                                                          ^
```